frontend/src/components/series/Series.js

```diff
@@ -0,0 +1,462 @@
+  fetchSeriesBySlug,
+  fetchSeriesInvitations,
+  fetchUser,
+  removeTeamSeriesRegistration
+} from "../../queries";
+import { displayDateShort } from "../../utils";
+import Info from "../alerts/Info";
+import Breadcrumbs from "../Breadcrumbs";
+import InvitationCard from "./Invitation";
+
+const TeamRow = props => {
+  const userIsTeamAdmin = () =>
+    props.adminTeams?.map(team => team.id).includes(props.team.id);
+
+  return (
+    <div class="flex items-center justify-between gap-x-4 py-4">
+      <div class="flex items-center gap-x-2">
+        <img
+          src={props.team.image ?? props.team.image_url}
+          class="mr-2 h-8 w-8 rounded-full"
+        />
+        <span class="font-semibold">
+          {props.team.name} {`(${props.regCount || "-"})`}
+        </span>
+        <Show when={userIsTeamAdmin()}>
+          <Icon class="h-4 w-4 text-blue-600" path={bolt} />
+        </Show>
+      </div>
+      <div class="justify-self-end">
+        <A
+          href={`/series/${props.seriesSlug}/team/${props.team.slug}`}
+          class={clsx(
+            "inline-flex items-center rounded-lg px-4 py-2 text-sm font-medium  focus:outline-none focus:ring-4",
+            userIsTeamAdmin()
+              ? "bg-blue-600 text-white hover:bg-blue-600 focus:ring-blue-300 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800"
+              : "border border-blue-700 bg-transparent text-blue-600  focus:ring-blue-300 dark:hover:bg-blue-700 dark:focus:ring-blue-800"
+          )}
+        >
+          <span class="inline-flex items-center gap-2">
+            <span>Roster</span>
+            <Icon path={arrowRight} class="inline h-3 w-3" />
+          </span>
+        </A>
+      </div>
+    </div>
+  );
+};
+
+const AdminTeamRow = props => {
+  return (
+    <div class="flex items-center justify-between gap-x-4 py-4">
+      <div class="flex items-center gap-x-2">
+        <img
+          src={props.team.image ?? props.team.image_url}
+          class="mr-2 h-8 w-8 rounded-full"
+        />
+        <span class="font-medium">{props.team.name}</span>
+      </div>
+      <div class="justify-self-end">
+        <Show when={props.registeredTeamIds.includes(props.team.id)}>
+          <button
+            type="button"
+            class={clsx(
+              "justify-self-end rounded-lg px-4 py-2 text-sm font-medium text-white focus:outline-none focus:ring-4",
+              "bg-red-500 hover:bg-red-600 focus:ring-red-300 dark:bg-red-600 dark:hover:bg-red-700 dark:focus:ring-red-800"
+            )}
+            onClick={() =>
+              props.deRegisterTeamMutation.mutate({
+                series_slug: props.seriesSlug,
+                body: {
+                  team_slug: props.team.slug
+                }
+              })
+            }
+          >
+            <span class="inline-flex items-center gap-2">
+              <Icon path={minus} class="inline h-3 w-3 text-white" />
+              <span>Remove</span>
```

Comment:
Lets remove removing from the series option? Cause now this will be used in Tournament reg and all and might cause issues if team accedentitaly removes.

Reply:
done